Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py

```diff
@@ -12,6 +12,9 @@
 # disable insecure warnings
 requests.packages.urllib3.disable_warnings()
 
+# Remove proxy if not set to true in params
+handle_proxy()
```

Comment:
why did we move it?

Reply:
to follow the new integration convention

Reply:
@yuvalbenshalom since when is it our convention?

Reply:
i also dont understand this, better to do it in the `try-except` block

Reply:
This is the change I have done.
Customer opened the issue attached.
We can decide to wait for his reply if you wish so.

Reply:
I Don't see how moving it will solve the bug, it does the exact same thing